src/components/TrackNewExercise.tsx

```diff
@@ -86,7 +97,7 @@ export function TrackNewExercise(props: TrackNewExerciseProps): JSX.Element {
     ].join('\n\n')
     const instructions = `${story}\n\n${tasks}`
 
-    fetch('https://exercism.io/git_api/concept_exercises', {
+    fetch(`${process.env.REACT_APP_EXERCISM_HOST}/git_api/concept_exercises`, {
```

Comment:
You can consider extracting this to a top-level constant, and giving a resolvable error if it's not present:

```
const EXERCISM_HOST: string | undefined = process.env.REACT_APP_EXERCISM_HOST
if (!EXERCISM_HOST) {
  throw new Error('REACT_APP_EXERCISM_HOST environment variable missing. Add it to a .env file to resolve this issue').
}

Reply:
Where should this top-level constant be defined? The URL is used in multiple places